docs/CHANGELOG.rst

```diff
@@ -6,6 +6,18 @@ All notable changes to this project will be documented in this file.
 The format is based on `Keep a Changelog <http://keepachangelog.com/>`_
 and this project adheres to `Semantic Versioning <http://semver.org/>`_.
 
+[3.1] - 2020-12-15
```

Comment:
I think it's non breaking right ? So it would be a minor release (we follow semver.org)